lib/asset_cloud.rb

```diff
@@ -1,31 +1,33 @@
-require 'addressable'
-require 'active_support'
+# frozen_string_literal: true
```

Comment:
Are we confident the test coverage would have caught any attempt to mutate frozen strings?

Reply:
(I see you already made some changes).